package.json

```
@@ -4,8 +4,9 @@
   "description": "",
   "main": "server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon server.js"
+    "test": "DOTENV_CONFIG_PATH=.env.test jest --setupFiles=dotenv/config",
+    "dev": "DOTENV_CONFIG_PATH=.env.dev nodemon server.js",
+    "start": "DOTENV_CONFIG_PATH=.env.prod node server.js"
```

```suggestion
    "dev": "DOTENV_CONFIG_PATH=.env.dev nodemon server.js",
    "start": "DOTENV_CONFIG_PATH=.env.prod nodemon server.js"
```

둘 다 nodemon으로 통합하는 방향을 추천드립니다